dradis/dradis-ce
# Form view missing fields

### Spec
The main problem here is that the parse_fields method does not recognize text that does not follow the field format or any text that is not preceded by a field header.

**Proposed solution**
The textile module should be able to recognize this format and add the necessary field in the inline view.


### How to test

1. Open an issue
2. Click the source view and add any text without any field headers
3. Click back on the inline view
4. Confirm that the text still appears.
